config/templates/cluster.yaml

```diff
@@ -53,15 +53,23 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # Number of controller nodes to create, for more control use `controller.autoScalingGroup` and do not use this setting
 #controllerCount: 1
 
-controller:
-  # Auto Scaling Group definition for controllers. If only `controllerCount` is specified, min and max will be the set to that value and `rollingUpdateMinInstancesInService` will be one less.
-  # autoScalingGroup:
-  #   minSize: 1
-  #   maxSize: 3
-  #   rollingUpdateMinInstancesInService: 2
+#controller:
+#  # Auto Scaling Group definition for controllers. If only `controllerCount` is specified, min and max will be the set to that value and `rollingUpdateMinInstancesInService` will be one less.
+#  autoScalingGroup:
+#    minSize: 1
+#    maxSize: 3
+#    rollingUpdateMinInstancesInService: 2
+#  privateSubnets:
+#    - availabilityZone: us-west-1a
+#      instanceCIDR: "10.0.4.0/24"
+#    - availabilityZone: us-west-1b
+#      instanceCIDR: "10.0.5.0/24"
 
 # Maximum time to wait for controller creation
```

Comment:
A comment specific to `controllerLoadBalancerPrivate` would be nice!
I'd imagine that something like `Set to true to put the ELB on the public subnet although having controller nodes on private subnets` would be enough but am open to your suggestions.